### PR TITLE
fix(argo-cd): Fix parser for podAntiAffinity in _common.tpl file

### PR DESCRIPTION
**Pull request by laiminhtrung1997**
Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [ ] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [ ] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [ ] Any new values are backwards compatible and/or have sensible default.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

Changes are automatically published when merged to `main`. They are not published on branches.

### CODE REVIEW

File: charts/argo-cd/templates/_common.tpl [hunk 1]
```diff
@@ -79,21 +79,15 @@ Node affinity
...
   {{- if (eq $preset.podAntiAffinity "soft") }}
+  preferredDuringSchedulingIgnoredDuringExecution:
   - weight: 100
     podAffinityTerm:
       labelSelector:
         matchLabels:
           app.kubernetes.io/name: {{ include "argo-cd.name" .context }}-{{ .component.name }}
       topologyKey: kubernetes.io/hostname
   {{- else }}
-  - weight: 100
```
Comment (pdrastil) [inline charts/argo-cd/templates/_common.tpl:91]: Why is preference for zone removed from hard affinity?

Comment (pdrastil) [reply charts/argo-cd/templates/_common.tpl:91]: This is taken from HA setup here: https://github.com/argoproj/argo-cd/blob/ba86be2fcbab6c53c311426a31d9761ff6934300/manifests/ha/base/overlays/argocd-repo-server-deployment.yaml#L10

Comment (laiminhtrung1997) [reply charts/argo-cd/templates/_common.tpl:91]: Dear @pdrastil 
I am truly sorry to bother you. It's just a mistake from me. I will close this PR.
Best regards.

Comment (pdrastil) [reply charts/argo-cd/templates/_common.tpl:91]: Hi @laiminhtrung1997 - no problem at all.

**Pull request closed without merging**